bpo-46399: Add GenericAlias support to `dict_keys`, `dict_values` and `dict_items` types (#30630)
python/cpython: Misc/NEWS.d/next/Library/2022-01-17-10-16-40.bpo-46399.8zNwq4.rst

> Implement :pep:`585` for :class:`types.DictKeysType`, :class:`types.DictValuesType` and :class:`types.DictItemsType`. Patch by Kumar Aditya.

Reviewer: ```suggestion
Add :meth:`~object.__class_getitem__` to :class:`types.DictKeysType`, :class:`types.DictValuesType` and :class:`types.DictItemsType`. Patch by Kumar Aditya.
```
